client/src/components/Upload/DragAndDropModal.vue

```diff
@@ -3,8 +3,10 @@ import { useFileDrop } from "composables/fileDrop";
 import { useGlobalUploadModal } from "composables/globalUploadModal";
 import { computed, ref } from "vue";
 
+import { Toast } from "@/composables/toast";
```

Comment:
Please use the composable export in composition components. The static object is for backwards compatibility, and we may want to remove it in the future, when moving away from bootstrap-vue

Reply:
@ElectronicBlueberry Clarifying question: in theory what you suggest makes sense. However, in practice, this is the [pass-through method](https://github.com/galaxyproject/galaxy/blob/dev/client/src/components/Toast.js) you want referenced instead...? I could only find it being referenced in `App.vue` or is that enough?

Reply:
@hujambo-dunia Not that Toast.js -- just use `useToast` from the composable here instad of the exposed `Toast` object.   That is only exposed for easier use in older options api components, while the function is more appropriate for composition api components.

Reply:
@dannon Thank you so much - that is helpful and now changes should be reflected.